### PR TITLE
Create golang PS server

### PR DESCRIPTION
**Pull request by skydoorkai**

### CODE REVIEW

File: elasticdl/pkg/server/psserver.go [hunk 1]
```diff
@@ -0,0 +1,86 @@
+package main
```
Comment (QiJune) [inline elasticdl/pkg/server/psserver.go]: package main --> package ps?

And what about 
elasticdl/pkg/server/psserver.go -->  elasticdl/pkg/ps/server.go

File: elasticdl/pkg/server/psserver_test.go [hunk 2]
```diff
@@ -0,0 +1,88 @@
+package main
```
Comment (QiJune) [inline elasticdl/pkg/server/psserver_test.go]: The same.

File: elasticdl/pkg/server/psserver.go (hunk 1 above)
Comment (skydoorkai) [reply elasticdl/pkg/server/psserver.go]: Renamed the directory and filename.

For the package name, it must be `main`, as this is an executable go file with `func main`.

**Review by QiJune — APPROVED** [review]
Excellent!

**Pull request merged**